packages/xo-server-web-hooks/src/index.js

```diff
@@ -2,16 +2,37 @@ import { createLogger } from '@xen-orchestra/log'
 
 const log = createLogger('xo:web-hooks')
 
+function constructPayload(isOfficeHook, data, type) {
+  if (isOfficeHook) {
+    // https://learn.microsoft.com/en-us/outlook/actionable-messages/message-card-reference
+    const facts = Object.keys(data).map(key => {
+      const value = data[key]
+      return { name: key, value: typeof value === 'string' ? value : JSON.stringify(value) }
+    })
+
+    return {
+      '@type': 'MessageCard',
+      '@context': 'https://schema.org/extensions',
+      themeColor: 'BE1621',
+      summary: 'New notification from the Xen-Orchestra webhook plugin',
+      sections: [{ title: `XO ${type.toUpperCase()} notification` }, { facts }],
+    }
+  }
+
+  return { ...data, type }
+}
```

Comment:
Since we want to support multiple formats, we might want to:
- Use an enum in the configuration
- Return a string here instead of an object (to be able to support non-JSON formats in the future)

What do you think?